src/Client.php

```diff
@@ -158,12 +160,12 @@ private function _raiseCurlError($errorNumber, $message) {
             case CURLE_COULDNT_CONNECT:
             case CURLE_COULDNT_RESOLVE_HOST:
             case CURLE_OPERATION_TIMEOUTED:
-                throw new Exception("Failed to connect to Killbill: " . $message);
+                throw new \Exception("Failed to connect to Killbill: " . $message);
```

Comment:
This is because of the usage of namespaces. If we'd just write `Exception` (without the backslash) here, the autloader would search for `Killbill\Client\Exception`.